app/javascript/src/infinite-scroll.js

```diff
@@ -66,6 +67,14 @@ function getInfiniteScrollContent(element) {
         element.innerHTML = "Load more"
         element.setAttribute("data-url", requestUrlString)
       }
+
+      // Initialize "Last updated" attribute in newly loaded items
+      const elements = document.querySelectorAll("[data-role~='timeago']")
```

Comment:
Perhaps instead of doing this here again, you could import our timeago.js file and call `render()` again (might need to import it as a nicer name)